src/handlers/health_check.rs

```diff
@@ -0,0 +1,36 @@
+use actix_web::{web, HttpResponse};
```

Comment:
This module (`handlers`) could live inside of the `server` module suggested above